### PR TITLE
Helm chart for kuberhealthy 0.1.1

### PR DESCRIPTION
**Pull request by integrii**
Signed-off-by: Eric Greer <eric.greer@comcast.com>

Satisfies https://github.com/Comcast/kuberhealthy/issues/26

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by integrii** [comment]
circleci above shows a generic error, but helm lint does in fact pass on this chart.

image

**Comment by integrii** [comment]
@unguiculus @seanknox May I have a review please? :-)

**Comment by integrii** [comment]
@unguiculus @seanknox Help please!  We are blocked on product release without this chart being approved.

File: stable/kuberhealthy/templates/clusterrolebinding.yaml
Comment (nvtkaszpir) [inline stable/kuberhealthy/templates/clusterrolebinding.yaml]: I suggest to fix missing newline.

File: stable/kuberhealthy/templates/clusterrole.yaml
Comment (nvtkaszpir) [inline stable/kuberhealthy/templates/clusterrole.yaml]: Please try to use v1  API if possible.

**Comment by integrii** [comment]
@nvtkaszpir I moved several things to the more current kubernetes versions . Thanks for pointing that out!
I also added a newline in that one file.

**Comment by integrii** [comment]
@nvtkaszpir Can you get some forward movement on publishing this for me?

**Comment by nvtkaszpir** [comment]
Unfortunately I have no power to push any PRs forward, you can ask owners for review on github/slack. Please to fix build issues first.

**Comment by nvtkaszpir** [comment]
Unfortunately I have no power to push any PRs forward, you can ask owners for review on github/slack. Please to fix build issues first, notice that maintainers must point to github accounts, unfortunately.

**Comment by integrii** [comment]
Thanks @nvtkaszpir.  The prior builds were erroring out, so the build that happened most recently was the first time I was actually able to see the errors.

I will work on those and then ping people on slack again.  Thank you!

**Comment by integrii** [comment]
@unguiculus and @seanknox can you please review my chart? :-)

**Comment by integrii** [comment]
@unguiculus and @seanknox help! Please review!

**Comment by unguiculus** [comment]
Please make sure you apply our guidelines (e. g. standard labels on all resources) and our RBAC best practices.

https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md
https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md

**Comment by integrii** [comment]
Thank you @unguiculus.  I have made the requested changes and now include all the recommended labels and variables.  I also added an `OWNERS` file and helpers for things like RBAC.  Is there anything else you would like to see changed?

 @seanknox 

Are you guys at kubecon this week?  I am there!

**Review by viglesiasce — CHANGES_REQUESTED** [review]
Remove PNG hinary

**Pull request closed without merging**
